IdentityServer4.Contrib.RedisStore/IdentityServer4.Contrib.RedisStore.csproj

```diff
@@ -29,8 +29,8 @@
 
   <ItemGroup>
     <PackageReference Include="IdentityServer4" Version="1.0.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
-    <PackageReference Include="StackExchange.Redis.StrongName" Version="1.2.4" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="StackExchange.Redis.StrongName" Version="1.2.6" />
```

Comment:
These is a reason why this library is stick to 1.2.4 version, please take a look at this issue #1

Reply:
Reverted the StackExchange.Redis.StrongName library update. Thank you for catching that!